app/policies/follow_request_policy.rb

```diff
@@ -0,0 +1,21 @@
+class FollowRequestPolicy < ApplicationPolicy
+  attr_reader :user, :follow_request
+
+  def index?
+    user.present?
+  end
+
+  def create?
+    user.present?
+
+  end
+
+  def destroy?
```

Comment:
Should all users be allowed to destroy or update follow requests? I'd think we'd only want the recipient of the request to update or destroy (accept or refuse).